R/data.R

```diff
@@ -69,17 +69,17 @@
 #'  \item{percasian}{Percent Asian.}
 #'  \item{percother}{Percent other races.}
 #'  \item{popadults}{Number of adults.}
-#'  \item{perchsd}{}
+#'  \item{perchsd}{Percent with high school diploma.}
 #'  \item{percollege}{Percent college educated.}
-#'  \item{percprof}{Percent profession.}
-#'  \item{poppovertyknown}{}
-#'  \item{percpovertyknown}{}
-#'  \item{percbelowpoverty}{}
-#'  \item{percchildbelowpovert}{}
-#'  \item{percadultpoverty}{}
-#'  \item{percelderlypoverty}{}
-#'  \item{inmetro}{In a metro area.}
-#'  \item{category}{}
+#'  \item{percprof}{Percent with professional degree.}
+#'  \item{poppovertyknown}{Population with known poverty status.}
+#'  \item{percpovertyknown}{Percent of population with known poverty status.}
+#'  \item{percbelowpoverty}{Percent of people below poverty line.}
+#'  \item{percchildbelowpovert}{Percent of children below poverty line.}
+#'  \item{percadultpoverty}{Percent of adults below poverty line.}
+#'  \item{percelderlypoverty}{Percent of elderly below poverty line.}
```

Comment:
`poppovertyknown` is always less than population total `poptotal`. So I personally inferred that this meant how many of these individuals do we actually know their poverty status for.

This is validated with some manual calculations to check if our manual calculation of percent poverty is equal to the one in the data set.

``` r
library(ggplot2)
library(dplyr)

data("midwest")

# Find number of counties our manual calculation fails
midwest %>%
  select(poptotal, poppovertyknown, percpovertyknown) %>%
  mutate(manual_percpoverty = poppovertyknown / poptotal * 100) %>%
  mutate(not_equal_per = !all.equal(percpovertyknown, manual_percpoverty)) %>%
  pull(not_equal_per) %>%
  sum()
#> [1] 0
```

<sup>Created on 2020-11-27 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

The remaining variables (`percbelowpoverty`, `percchildbelowpovert`, `percadultpoverty`, and `percelderlypoverty`) were my own personal inferences on the pieces of information based on the variable name itself.